server/src/Config.hs

```diff
@@ -59,6 +61,11 @@ configParser environment = Config
                   metavar "HOST:PORT" <>
                   help "If provided, Icepeak collects various metrics and provides them to Prometheus on the given endpoint."
                  ))
+  <*> fmap (fromMaybe 256)
+           (optional (option auto
+                       (long "queue-capacity" <>
+                        metavar "INTEGER" <>
+                        help "Higher values allow more requests to come in within a short amount of time.")))
```

Comment:
Maybe mention that this is a trade-off between the amount of data risked losing, the ability to accept bursts of requests without denying them.

Reply:
Good point. :+1: